.nuget/NuGet.targets

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
In order to NOT use the msbuild integrated approach, this file has to reside somewhere else, e.g. as `build/NuGet.targets`